datasets/aardgasverbruik/aardgasverbruik_mra_liander/v1.0.0.json

```diff
@@ -0,0 +1,146 @@
+{
+  "id": "aardgasverbruik_mra_liander",
+  "title": "Data over Standaard Jaarverbruik (SJV) en aantal aansluitingen per postcode-range in de MRA, vanaf 2018. Nieuwe data aan het einde van de maand.",
+  "type": "table",
+  "version": "1.0.0",
+  "schema": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "identifier": "id",
+    "type": "object",
+    "additionalProperties": false,
+    "required": [
+      "id",
+      "schema"
+    ],
+    "display": "id",
+    "properties": {
+      "schema": {
+        "$ref": "https://schemas.data.amsterdam.nl/schema@v1.1.1#/definitions/schema"
+      },
+      "id": {
+        "title": "id",
+        "type": "string",
+        "description": "Unieke identificatie"
+      },
+      "dataDate": {
+        "title": "dataDate",
+        "type": "date",
+        "description": "Datum van de aangeleverde data"
+      },
+      "postcodeVanaf": {
+        "title": "postcodeVanaf",
+        "type": "string",
+        "description": "Postcode vanaf (eerste postcode in postcoderange)"
+      },
+      "postcodeTotEnMet": {
+        "title": "postcodeTotEnMet",
+        "type": "string",
+        "description": "Postcode t/m (laatste postcode in postcoderange)"
+      },
+      "maand": {
+        "title": "maand",
+        "type": "integer",
+        "jaar": "Maand waarin voorspelling standaard jaarverbruik is gedaan"
+      },
+      "jaar": {
+        "title": "jaar",
+        "type": "integer",
+        "jaar": "Jaar waarin voorspelling standaard jaarverbruik is gedaan"
+      },
+      "aantalEans": {
+        "title": "aantalEans",
+        "type": "integer",
+        "description": "Aantal aansluitingen"
+      },
+      "sjv": {
```

Comment:
Ook hier de het id liever uitschrijven en evt. een shortname voor gebruik in de database.

Reply:
Aangepast naar standaardJaarverbruik